common-rendering/src/components/figCaption.tsx

```diff
@@ -16,12 +16,18 @@ import { fill, text } from '@guardian/common-rendering/src/editorialPalette';
 
 // ----- Sub-Components ----- //
 
-interface CameraProps {
+enum Variant {
```

Comment:
Is `Variant` perhaps quite a generic term? Could we make this more specific by calling it something like `Icon` or `IconVariant`?